lib/slack/schedule-manager.ts

```
@@ -0,0 +1,151 @@
+
+        if (isNaN(hours) || isNaN(minutes) || hours < 0 || hours > 23 || minutes < 0 || minutes > 59) {
+            console.warn(`Invalid time format: ${time}, using default 10:00`);
+            return `0 10 * * *`;
+        }
+
+        switch (integration.frequency) {
+            case 'daily':
+                return `${minutes} ${hours} * * *`;
+
+            case 'weekly':
+                const weeklyDay = integration.weeklyDay || 'monday';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)